src/DebugConfigurationProvider.ts

```diff
@@ -63,7 +63,7 @@ export class DebugConfigurationProvider implements vscode.DebugConfigurationProv
     }
 
     const testCommand = folder && getTestCommand(folder.uri.fsPath)
-    if (isCRATestCommand(testCommand)) {
+    if (isCreateReactAppTestCommand(testCommand)) {
```

Comment:
All this talk about improving maintainability made me remember just how long it took to figure out what "CRA" was from reading the issues. Let's spell it out for folks.